x-pack/plugins/lens/public/app_plugin/lens_top_nav.tsx

```diff
@@ -163,7 +163,7 @@ function getLensTopNavConfig(options: {
 
   if (actions.getUnderlyingDataUrl.visible) {
     const exploreDataInDiscoverLabel = i18n.translate('xpack.lens.app.exploreDataInDiscover', {
-      defaultMessage: 'Explore data in Discover',
+      defaultMessage: 'Explore in Discover',
```

Comment:
@cqliu1 I personally don't mind this change, but has it been reviewed with the docs team? I believe they supplied the original wording.

Reply:
Yes, @amyjtechwriter made the suggestion for the copy update

Reply:
Cool. Sorry, I missed that comment!